afs/afs-core/src/test/java/com/powsybl/afs/ProjectTest.java

```diff
@@ -0,0 +1,51 @@
+package com.powsybl.afs;
```

Comment:
You forget to add the license information. Change your IDE configuration not to forget it